src/App.jsx

```diff
@@ -36,6 +39,30 @@ const PageContainer = ({ children }) => {
 };
 
 function MainContent() {
+	const [cookies] = useCookies(['session'])
+	const cookiesInfo = JSON.parse(localStorage.getItem(API_CONFIG.SESSION_DATA.COOKIE));
+	const { user, setUserData } = useContext(GlobalDataContext);
+
+	// check if cookie is expired
+	if (!user) {

```

Comment:
This will be True if user = null.

Reply:
that's expected, in the context initially the userData is null, then set to an emtpy object in case the cookie is expired.